### ISSUE TITLE
404：not found

### ISSUE THREAD

**Issue by txl121** [tor2web/Tor2web#374]
![123](https://user-images.githubusercontent.com/41889969/99221822-4662af80-281c-11eb-8c18-b36f7ea28eca.PNG)

**Comment by darth-veitcher** [comment]
Yes the installer seems to be non-existent... whats the purpose of linking elsewhere to a script for installation when it could just be included within the repository itself?

**Comment by darth-veitcher** [comment]
Is this project abandoned now?

**Comment by evilaliv3** [comment]
Thank you James,

The project is not abandoned but unfortunately lately i did not find ways
to move it forward.

I will try to make a new package for the current releases of Debian/Ubuntu.

Which is your interest in the project? Would you like to contribute with
some development?

**Comment by darth-veitcher** [comment]
Thanks for the response, appreciate that there's probably plenty of competing priorities.

More than happy to help out if I can as this feels like a valuable project to keep alive for the community.

My interest right now is identifying whether it's possibly to provide general purpose proxying from clearnet to onion services and I'm happy to host some of these on my own infrastructure.

To begin with I was considering looking at some kind of onion-shortening service as an interesting experiment and potential use case.

**Comment by darth-veitcher** [comment]
Running through the installation script today, will update here with progress unless you'd prefer new issues @evilaliv3 ?

So far hit a GPG error when I've cloned the repo and attempted to run the installation script.

```bash
Checking preliminary Tor2web requirements
 + apt-key requirement meet
 + apt-get requirement meet
Detected OS: Ubuntu - focal, which is not supported: fallback to 'buster'
Adding GlobaLeaks PGP key to trusted APT keys
Running: "apt-key add /globaleaks_key"... SUCCESS
Running: "rm -f /globaleaks_key"... SUCCESS
Running: "apt-get update -y"... FAIL
Ouch! The installation failed.
COMBINED STDOUT/STDERR OUTPUT OF FAILED COMMAND:
Hit:2 http://nova.clouds.archive.ubuntu.com/ubuntu focal InRelease
Hit:3 http://nova.clouds.archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:4 http://nova.clouds.archive.ubuntu.com/ubuntu focal-backports InRelease
Hit:5 https://download.docker.com/linux/ubuntu focal InRelease
Ign:1 https://deb.globaleaks.org buster/ InRelease
Get:6 https://deb.globaleaks.org buster/ Release [744 B]
Get:7 https://deb.globaleaks.org buster/ Release.gpg [833 B]
Hit:8 http://security.ubuntu.com/ubuntu focal-security InRelease
Ign:7 https://deb.globaleaks.org buster/ Release.gpg
Reading package lists...
W: GPG error: https://deb.globaleaks.org buster/ Release: The following signatures were invalid: BADSIG 32E6792624045008 GlobaLeaks software signing key <info@globaleaks.org>
E: The repository 'http://deb.globaleaks.org buster/ Release' is not signed.
```

**Comment by wuladapang** [comment]
![1678024801328](https://user-images.githubusercontent.com/53263049/222965038-6fae0d4f-d21c-4721-bfe8-04a5983dfb11.png)
I run the install.sh, but this process is wrong.
Does anyone know how to solve? Thanks

**Comment by EsmailELBoBDev2** [comment]
> ![1678024801328](https://user-images.githubusercontent.com/53263049/222965038-6fae0d4f-d21c-4721-bfe8-04a5983dfb11.png) I run the install.sh, but this process is wrong. Does anyone know how to solve? Thanks

H, I know I'm late but. the script did not include a URL to import PGP keys for the package (hardned it) so I advice you to go to https://deb.globaleaks.org/ and download the "gloaballeaks" key and edit it manually

**Comment by CloudWaddie** [comment]
Try using my install script.
https://github.com/CloudWaddie/tor2web-install-script
